__tests__/Case-test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Case} from '../src/containers/Case/Case';
+import { mountWithIntl, shallowWithIntl } from '../src/helpers/intl-enzyme-test-helper.js';
+import caseData from './case_data.json';
+
+function setup() {
+  const props = {
+    intl: {
+        formatDate: jest.fn(),
+        formatTime: jest.fn(),
+        formatRelative: jest.fn(),
+        formatNumber: jest.fn(),
+        formatPlural: jest.fn(),
+        formatMessage: jest.fn(),
+        formatHTMLMessage: jest.fn(),
+        now: jest.fn(),
+    },
+    params: {nodeID: 468}
+  }
+
+  const enzymeWrapper = shallowWithIntl(<Case {...props} />)
+  return {
+    props,
+    enzymeWrapper
+  }
+}
+
+describe('containers', () => {
+  describe('Case', () => {
+    it('should render proper data for case', () => {
+
+      const { enzymeWrapper } = setup()
+      enzymeWrapper.setState({data: caseData})
+      expect(enzymeWrapper.find('.sub-heading').length).toBe(2);
+      expect(enzymeWrapper.find('h2.case-title').text()).toBe("Agenda 21 Locale (Province of Forli-Cesena, Italy)");
```

Comment:
This (and the line above) feels like it could be a very brittle test.  OK for now, but we may want to generalize, and/or at some point test against a database which is known/fixed.